Cesium.Preprocessor/CPreprocessor.cs

```diff
@@ -284,6 +298,12 @@ await foreach (var item in subProcessor.GetPreprocessingResults())
         yield return new Token<CPreprocessorTokenType>(new Range(), new Location(), "\n", NewLine);
     }
 
+    private static IEnumerable<IToken<CPreprocessorTokenType>> TokenizeString(string code)
+    {
+        var tokenizer = new CPreprocessorLexer("c:/null.c", code);
```

Comment:
Let's either mark this as a `null` path (making the parameter nullable), or use some more specific name, like `<null>` or something.